test/e2e/storage/drivers/in_tree.go

```diff
@@ -1219,7 +1250,7 @@ func (g *gcePdDriver) CreateVolume(volType testpatterns.TestVolType) interface{}
 	if volType == testpatterns.InlineVolume {
 		// PD will be created in framework.TestContext.CloudConfig.Zone zone,
 		// so pods should be also scheduled there.
-		g.driverInfo.Config.NodeSelector = map[string]string{
```

Comment:
Can some fields in `VolumeTestConfig` be removed (e.g. `NodeSelector`)?

Reply:
The GCE PD driver relies on `NodeSelector` and I was also using it out-of-tree, so no, I don't think it can be removed.